src/components/HomePage/SearchArea/SearchBarTheme.js

```diff
@@ -0,0 +1,32 @@
+import AppTheme, { darkTextColor } from '../../../AppTheme';
+
+export default ({
+   
+    root: {
+        width: 200,
+    },
+
+    cssLabel: {
+        color: darkTextColor,
+    },
+    cssLabelShrink: {},
+    cssFocused: {},
+    cssOutlinedInput: {
+        
+        '& $notchedOutline': {
+            borderColor: darkTextColor,
+        },
+
+        '&$cssFocused $notchedOutline': {
+            borderColor: AppTheme.palette.primary.main,
+        },
+    },
+    notchedOutline: {},
+    formControl: {},
+    
+
+    searchButton: {
+        color: darkTextColor,
+    },
+});
+  
```

Comment:
Themes inside themes? Can't this all be set in the global theme and passed to everyone that wants it using a `ThemeProvider`? Would simplify things as then it would be as simple as using `makeStyles` or `useTheme` for everything, iirc 😃

Reply:
This theme is specific to a component, that way it should not be in a global place

Reply:
Misread this, this is the argument of `makeStyles` for the `SearchBar` it seems.
As I've said further down somewhere, I'm in favor of keeping this in the component to not lose scope, despite possible increased code clutter.

Reply:
These styles can get quite big for trickier customizations, thus we should keep them separated to avoid doing all the work later when some component requires 100 lines of styling and upon separating it, we realize it becomes inconsistent

Reply:
I'm okay with this, will keep this in mind and see the direction to take in the future. Will need to keep an eye for file clutter for smaller components basically.